Source/engine/render/scrollrt.cpp

```diff
@@ -664,17 +664,8 @@ void DrawMonsterHelper(const Surface &out, Point tilePosition, Point targetBuffe
 	const ClxSprite sprite = monster.animInfo.currentSprite();
 
 	Displacement offset = monster.getRenderingOffset(sprite);
-	if (monster.isWalking()) {
-		bool isSideWalkingToLeft = monster.mode == MonsterMode::MoveSideways && monster.direction == Direction::West;
-		if (isNegativeMonster && !isSideWalkingToLeft)
-			return;
-		if (!isNegativeMonster && isSideWalkingToLeft)
-			return;
-		if (isSideWalkingToLeft)
-			offset -= Displacement { 64, 0 };
-	} else if (isNegativeMonster) {
+	if (isNegativeMonster)

```

Comment:
This is now always done. Please move it to the very top of the function and remove the other case.
```cpp
if (mi < 0)
		return;
```